shared/chat/conversation/messages/react-button/emoji-row/index.js

```diff
@@ -94,6 +100,12 @@ class EmojiRow extends React.Component<Props, {showingPicker: boolean}> {
 
 const styles = Styles.styleSheetCreate({
   container: Styles.platformStyles({
+    common: {
+      paddingBottom: Styles.globalMargins.xtiny,
```

Comment:
:sheep: you can use `...Styles.padding(Styles.globalMargins.xtiny, Styles.globalMargins.xsmall)`